plan/logical_plan_builder.go

```diff
@@ -1341,22 +1374,28 @@ func (b *planBuilder) unfoldWildStar(p LogicalPlan, selectFields []*ast.SelectFi
 		}
 		dbName := field.WildCard.Schema
 		tblName := field.WildCard.Table
+		findTblNameInSchema := false
 		for _, col := range p.Schema().Columns {
 			if (dbName.L == "" || dbName.L == col.DBName.L) &&
 				(tblName.L == "" || tblName.L == col.TblName.L) &&
 				col.ID != model.ExtraHandleID {
+				findTblNameInSchema = true
 				colName := &ast.ColumnNameExpr{
 					Name: &ast.ColumnName{
 						Schema: col.DBName,
 						Table:  col.TblName,
 						Name:   col.ColName,
 					}}
+				ast.SetFlag(colName)
```

Comment:
Why do we need to set flag for column name ? I grep the whole code. It seems there is the only place that sets flag